keras_cv/models/backbones/resnet_v2/resnet_v2_backbone.py

```diff
@@ -217,6 +217,8 @@ def apply_stack(
             stack. Use "basic_block" for ResNet18 and ResNet34.
         first_shortcut: bool. Use convolution shortcut if `True` (default),
             otherwise uses identity or pooling shortcut, based on stride.
+        stack_index: int, the index of this stack in the ResNet model. Defaults
```

Comment:
the use case of this parameter is not clear to me.  

I think thats a good sign that we should add some information in the docstring to clarify how this impacts end users?

Reply:
I think its used to generate the default name only. We could do this inside the loop instead. Let me know what you think